packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -312,6 +313,9 @@ const styles = StyleSheet.create({
         // Set the height to remove the padding from buttons
         height: 18,
     },
+    link: {
+        fontSize: 14,
```

Comment:
**thought:** This it unrelated to your specific change. But I can see how we can create design tokens for typography styles so we could reuse those values for these kind of situations.

```
e.g.
fontSize: tokens.typography.small
````

Hopefully, we'll get there at some point where we can centralize our tokens into a single place (e.g. colors, typography, spacing as part of the same object), so we can leverage theming in a more approachable way (cc @megaroeny).